docs/api/GetParam.md

```diff
@@ -71,6 +71,36 @@ Sample of double-call:
     }
 ```
 
+# Special Parameters
+
+## QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES
+
+Returns an array of well-known sizes (in bytes) for each version of the `QUIC_STATISTICS_V2` struct. This allows applications to determine the correct buffer size for statistics queries, even as new versions are added in future MsQuic releases.
```

Comment:
I think it would be useful to explain when this is needed: this isn't useful for an app that just want access to the stats, but only for a client that wants to forward the stats to a final consumer that will be aware of the new size.

For most app, getting the size directly from the header constant should be enough and they should not overthink it.